LayoutTests/fast/dom/DOMURL/parsing-expected.txt

```diff
@@ -13,7 +13,7 @@ PASS breakDownURL('file:///Users/darin') is 'protocol=file:, pathname=/Users/dar
 PASS breakDownURL('data:text/html,<b>foo</b>') is 'protocol=data:, pathname=text/html,<b>foo</b>, origin=null'
 PASS breakDownURL('http://a:b@c:1/e/f?g%h') is 'protocol=http:, username=a, password=b, hostname=c, host=c:1, port=1, pathname=/e/f, search=?g%h, origin=http://c:1'
 PASS breakDownURL('http://ex%61mple.com/') is 'protocol=http:, host=example.com, pathname=/, origin=http://example.com, toString=http://example.com/'
-PASS breakDownURL('http://ex%2fmple.com/') threw exception TypeError: Type error.
+PASS breakDownURL('http://ex%2fmple.com/') threw exception TypeError: "http://ex%2fmple.com/" cannot be parsed as a URL..
```

Comment:
The double dot looks a bit funny here, but I guess that's normal?

Reply:
the first dot comes from the error message, and probably the second dot comes from the test. Let see what is the test. 

The second dot is coming from the test framework

https://github.com/WebKit/WebKit/blob/22a32809f2d43aa8c6984784fe2a53078e43a47f/LayoutTests/resources/js-test.js#L709

Reply:
Does that mean, error messages should not have dots.

Reply:
I will merge as-is.